docs-source/content/userguide/managing-domains/configoverrides/_index.md

```diff
@@ -36,22 +36,22 @@ You can use overrides to customize domains as they are moved from QA to producti
 
 * Make sure your domain home meets the prerequisites. See [Prerequisites](#prerequisites).
 * Make sure your overrides are supported. See [Typical overrides](#typical-overrides) and [Unsupported overrides](#unsupported-overrides).
-* Create a Kubernetes configuration map that contains:
+* Create a Kubernetes ConfigMap that contains:
```

Comment:
The config overrides documentation needs to be updated to reflect the new introspectVersion support. This includes, but may not be limited to:

- Adding a section about introspectVersion and/or reference the new related documentation.
- Looking for keywords like stopping, starting, and restarting and make it clear in their respective locations that a full domain restart isn't always needed.
- Updating the 'internal design flow' section at the end.

Reply:
It looks like the 'checklist', 'step-by-step', and 'limitations' sections could still use some rework.  Perhaps others?